protostar/cairo/short_string.py

```diff
@@ -0,0 +1,8 @@
+CairoShortString = int
+
+
+def short_string_to_str(value: CairoShortString) -> str:
```

Comment:
Doesn't starknet expose any function for that 🤔

Reply:
no unfortunately only the inverse